src/reducers/singleArticle.reducer.js

```diff
@@ -34,6 +40,15 @@ const singleArticle = (state = initialState, action) => {
         ...state,
         error: false,
       };
+    case actionTypes.LIKE_SUCCESS:
+      return {
+        ...state,
+        article: updateArticleLike(state.article, action.likeCount),
```

Comment:
Rename this to `updateArticleLikes` @Akinmyde

Reply:
Also the first argument should be `state.articles` since we are going to have more than 1 article in the array. 

Also why are we not using the `articleId` to find the article to be updated like we agreed? @Akinmyde

Reply:
We only have one article in the state @rotimi-babalola